lib/sdam/topology_description.js

```diff
@@ -272,7 +273,14 @@ class TopologyDescription {
   }
 }
 
-function topologyTypeForServerType(serverType) {
+function topologyTypeForServerType(serverType, options) {
+  if (serverType === ServerType.Standalone) return TopologyType.Single;
+  if (serverType === ServerType.Unknown && options.directConnection === false) {
```

Comment:
it looks like this can be combined with the line below:
```
if (options.directConnection && (serverType === ServerType.Unknown || serverType === ServerType.RSGhost)) {
   ...
}
```

Reply:
👍